auth_signup_verify_email/controllers/main.py

```diff
@@ -9,7 +9,8 @@
 _logger = logging.getLogger(__name__)
 
 try:
-    from email_validator import validate_email, EmailSyntaxError
+    from email_validator import validate_email, EmailSyntaxError, \
+    EmailUndeliverableError
```

Comment:
Indent this line and declare a dummy replacement for the cases when `validate_email` is used instead, as you can see below, to add the backwards compatibility support.

Reply:
Hi @Yajo 

Thank you for the review. I have already applied the changes you asked, please review them and let me know if there is something else needed.

Best Regards.